executions/graphql-kotlin-dataloader-instrumentation/src/main/kotlin/com/expediagroup/graphql/dataloader/instrumentation/syncexhaustion/state/SyncExecutionExhaustedState.kt

```diff
@@ -51,9 +49,9 @@ class SyncExecutionExhaustedState(
      */
     fun beginExecuteOperation(
         parameters: InstrumentationExecuteOperationParameters
-    ): InstrumentationContext<ExecutionResult> {
+    ): InstrumentationContext<ExecutionResult>? {
         executions[parameters.executionContext.executionInput] = ExecutionBatchState()
```

Comment:
should this be wrapped in compute if abstent?

Reply:
ill add it just in case to prevent any possible race condition